src/envs/painting.py

```diff
@@ -524,37 +523,28 @@ def _Pick_policy(self, state: State, memory: Dict,
         obj_x = state.get(obj, "pose_x")
         obj_y = state.get(obj, "pose_y")
         obj_z = state.get(obj, "pose_z")
-        dx, dy, dz, grasp = params
-        arr = np.array(
-            [obj_x + dx, obj_y + dy, obj_z + dz, grasp, 1.0, 0.0, 0.0, 0.0],
-            dtype=np.float32)
-        # The addition of dx, dy, and dz could cause the action to go
-        # out of bounds, so we clip it back into the bounds for safety.
+        grasp, = params
+        arr = np.array([obj_x, obj_y, obj_z, grasp, 1.0, 0.0, 0.0, 0.0],
+                       dtype=np.float32)
+        # The grasp could cause the action to go out of bounds, so we clip
```

Comment:
why was this not a thing before?

Reply:
it was. we were already clipping before too